lib/datasource/rubygems/releases.js

```diff
@@ -0,0 +1,28 @@
+const { nonEmptyArray } = require('@sindresorhus/is');
+const { getDependency } = require('./get');
+
+const RUBYGEMS_URL = 'https://rubygems.org';
+
+const getRegistries = (envSources, configSources) =>
+  Array.prototype.concat.call(envSources || configSources || RUBYGEMS_URL);
+
+async function getPkgReleases({ fullname: dependency }, config = {}) {
+  const { registryUrls, compatibility = {} } = config;
+
+  const envSources = process.env.RUBYGEMS_URL ? process.env.RUBYGEMS_URL : null;
```

Comment:
Is this standardised anywhere by Bundler?

Reply:
Nope, i did it to analogy with [pypi datasource](https://github.com/renovatebot/renovate/blob/master/lib/datasource/pypi/index.js#L29-L46)

Reply:
I think that one has a convention in pip so that's why it's included. In this case, let's leave it out, because users can define `RENOVATE_REPOSITORY_URLS` anyway.